src/linker/Linker/BCL.cs

```diff
@@ -71,6 +71,10 @@ public static TypeDefinition FindPredefinedType (string ns, string name, LinkCon
 			if (cache.TryGetValue ("System.Private.CoreLib", out corlib))
 				return corlib.MainModule.GetType (ns, name);
 
+			// System.NotSupportedException lives here on .netcore
```

Comment:
fyi, I had to add this so that `System.NotSupportedException` could be found.  This could become a tedious pattern to maintain